src/StaticFAQ/SearchAllFAQs.js

```diff
@@ -90,20 +90,16 @@ class SearchAllFAQs extends React.Component<Props> {
     }
     if (props) {
       const edges = idx(props, _ => _.allFAQs.edges) || [];
-      const results = edges.map(edge => edge.node.results);
-      return this.renderFAQs(results);
+      const faqList = edges.map(edge => edge.node);
+      return this.renderFAQs(faqList);
     }
     return <Loader />;
   };
 
-  renderFAQs = (results: FAQs[]) => (
+  renderFAQs = (faqList: FAQs[]) => (
     <div className="scrollable-box">
-      {results.map(result => (
-        <FAQArticle
-          key={result.articleId}
-          title={result.title}
-          perex="How to find your best deal fast and easy"
-        />
+      {faqList.map(faq => (
+        <FAQArticle key={faq.id} title={faq.title} perex={faq.perex} />
```

Comment:
this is not sufficient change, please use `FAQArticle` with relay fragments & remove redundant `FAQArticle` in this file